src/main/java/org/team1540/robot2025/subsystems/grabber/GrabberIO.java

```diff
@@ -0,0 +1,23 @@
+package org.team1540.robot2025.subsystems.grabber;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface GrabberIO {
+    @AutoLog
+    class GrabberIOInputs {
```

Comment:
You should log the connection status of the motor and the CANDi, and have dashboard alerts when they get disconnected.